recipe/bld.bat

```diff
@@ -8,7 +8,7 @@ if not "%CUDA_COMPILER_VERSION%" == "None" (
     if "%CUDA_COMPILER_VERSION%" == "12.0" (
         set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%CUDA_COMPILER_VERSION%
         set CUDA_BIN_PATH=%CUDA_PATH%\bin
-        set "CUDA_TOOLKIT_INCLUDE_DIR=%CUDA_PATH%\include;%PREFIX%\Library\include;"
+        set CUDA_TOOLKIT_INCLUDE_DIR=%PREFIX%\Library\include
```

Comment:
It seems to me strange that `nvcc` would react to a variable that starts with `CUDA_TOOLKIT_` prefix. Did you check if by any chance it reacts to `CPATH` or `CPLUS_INCLUDE_PATH` (as gcc, see https://gcc.gnu.org/onlinedocs/cpp/Environment-Variables.html) or `INCLUDE` (used by MSVC, I could not find any docs but if I recall correctly it is used).

Reply:
It should react because I included it as a search path for `nvcc` in https://github.com/conda-forge/warp-lang-feedstock/blob/41f0394a6d72afb39c2e42db8d00938648f83df1/recipe/04-find_nvcc_env.patch#L44

I can try to use the options you mentioned instead

Reply:
ahhhhhhhh I missed that.